Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -12,33 +12,44 @@ protocol ServiceRunnable {
     ///   - body: The contents of the body of the request.
     func post<T: Encodable, U: Decodable>(to path: PathType, with body: T) -> AnyPublisher<U, Error>
 
+    /// Get service call to a particular path
+    /// - Parameters:
+    ///   - path: Endpoint to execute the GET call.
+    func get<U: Decodable>(to path: PathType) -> AnyPublisher<U, Error>
+
     /// PUT service call to a particular path with a body.
     /// - Parameters:
     ///   - data: Data to be uploaded
     ///   - url: Endpoint to upload to
     ///   - restMethod: The rest method to be used (PUT, POST etc )
     func upload(data: Data,
                 to url: String,
-                with restMethod: RestMethod) -> AnyPublisher<UploadResponse, Error>}
+                with restMethod: RestMethod) -> AnyPublisher<UploadResponse, Error>
+}
 
 extension ServiceRunnable {
-
     var baseURL: URL? {
         if SmileID.useSandbox {
             return URL(string: SmileID.config.testLambdaUrl)
         }
         return URL(string: SmileID.config.prodLambdaUrl)
-
     }
 
     func post<T: Encodable, U: Decodable>(to path: PathType, with body: T) -> AnyPublisher<U, Error> {
         return createRestRequest(path: path,
                                  method: .post,
                                  headers: [.contentType(value: "application/json")],
-                                 body: body
-        )
-        .flatMap(serviceClient.send)
-        .eraseToAnyPublisher()
+                                 body: body)
+            .flatMap(serviceClient.send)
+            .eraseToAnyPublisher()
+    }
+
+    func get<U: Decodable>(to path: PathType) -> AnyPublisher<U, Error> {
+        return createRestRequest(path: path,
+                                 method: .get,
+                                 headers: [.contentType(value: "application/json")])
```

Comment:
GET requests don't have a request body, therefore the content type header should not be set